src/test/java/org/gridsuite/modification/server/modifications/DeleteAttachingLineTest.java

```diff
@@ -0,0 +1,121 @@
+import org.gridsuite.modification.server.ModificationType;
+import org.gridsuite.modification.server.NetworkModificationException;
+import org.gridsuite.modification.server.dto.DeleteAttachingLineInfos;
+import org.gridsuite.modification.server.dto.ModificationInfos;
+import org.gridsuite.modification.server.utils.MatcherModificationInfos;
+import org.gridsuite.modification.server.utils.NetworkWithTeePoint;
+import org.junit.Test;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import java.util.UUID;
+
+import static org.gridsuite.modification.server.NetworkModificationException.Type.DELETE_ATTACHING_LINE_ERROR;
+import static org.gridsuite.modification.server.utils.MatcherDeleteAttachingLineInfos.createMatcherDeleteAttachingLineInfos;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+/**
+ * @author bendaamerahm <ahmed.bendaamer at rte-france.com>
+ */
+public class DeleteAttachingLineTest extends AbstractNetworkModificationTest {
+
+    @Override
+    protected Network createNetwork(UUID networkUuid) {
+        return NetworkWithTeePoint.create(networkUuid);
+    }
+
+    @Override
+    protected ModificationInfos buildModification() {
+        return DeleteAttachingLineInfos.builder()
+                .type(ModificationType.DELETE_ATTACHING_LINE)
+                .lineToAttachTo1Id("l1")
+                .lineToAttachTo2Id("l2")
+                .attachedLineId("l3")
+                .replacingLine1Id("replacingLineId")
+                .replacingLine1Name("replacingLine")
+                .build();
+    }
+
+    @Override
+    protected ModificationInfos buildModificationUpdate() {
+        return DeleteAttachingLineInfos.builder()
+                .type(ModificationType.DELETE_ATTACHING_LINE)
+                .lineToAttachTo1Id("l1")
+                .lineToAttachTo2Id("l2")
+                .attachedLineId("l3")
+                .replacingLine1Id("replacingLineIdEdited")
+                .replacingLine1Name("replacingLine")
+                .build();
+    }
+
+    @Override
+    protected MatcherModificationInfos createMatcher(ModificationInfos modificationInfos) {
+        return createMatcherDeleteAttachingLineInfos((DeleteAttachingLineInfos) modificationInfos);
+    }
+
+    @Override
+    protected void assertNetworkAfterCreation() {
+        assertNull(getNetwork().getLine("l1"));
+        assertNull(getNetwork().getLine("l2"));
+        assertNotNull(getNetwork().getLine("replacingLineId"));
+    }
+
+    @Override
+    protected void assertNetworkAfterDeletion() {
+        assertNotNull(getNetwork().getLine("l1"));
+        assertNotNull(getNetwork().getLine("l2"));
+        assertNull(getNetwork().getLine("replacingLineIdEdited"));
```

Comment:
This assert is symmetrical with assertNetworkAfterCreation: I think we should test assertNull on line "replacingLineId"